.coderabbit.yaml

```
@@ -0,0 +1,141 @@
+    - path: "**/packages/*/src/*.tsx"
+      instructions: |
+        Review React component code for:
+        - Consistent component patterns following existing components (forwardRef, props interface, const enums)
+        - Proper TypeScript usage with strict typing
+        - Accessibility best practices (semantic HTML, ARIA attributes)
+        - Component composition patterns using Radix UI Slot when applicable
+        - ThemeProvider integration - components should work with all theme variants (1st-4th)
+        - Consistent prop naming and structure with existing components
+        - Proper use of clsx for conditional className handling
+        - Integration with design token system and theme contract variables
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)